### PR TITLE
Bug 1745877: Fix typo in dashboard name

### PR DESCRIPTION
**Pull request by paulfantom**
Update kubernetes-mixin to fix dashboard typo.

Additionally this fixes problematic alert.

/cc @LiliC @s-urbaniak

### CODE REVIEW

File: assets/prometheus-k8s/rules.yaml [hunk 1]
```diff
@@ -710,7 +710,7 @@ spec:
       annotations:
         message: Job {{ $labels.namespace }}/{{ $labels.job_name }} failed to complete.
       expr: |
-        kube_job_status_failed{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kube-state-metrics"}  > 0
+        kube_job_failed{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kube-state-metrics"}  > 0
```
Comment (s-urbaniak) [inline assets/prometheus-k8s/rules.yaml]: this doesn't have any side effect for us?

**Comment by s-urbaniak** [comment]
small question, else lgtm!

File: assets/prometheus-k8s/rules.yaml (hunk 1 above)
Comment (paulfantom) [reply assets/prometheus-k8s/rules.yaml]: I didn't find any.

Comment (LiliC) [reply assets/prometheus-k8s/rules.yaml]: I found something in openshift-azure, but no clue if these files are used, looks generated https://github.com/openshift/openshift-azure/blob/48f07ce4f72307ff4397eb5d2d9352fbee6597b6/pkg/entrypoint/metricsbridge/metrics-bridge-example-config.yaml#L46

Comment (paulfantom) [reply assets/prometheus-k8s/rules.yaml]: Yeah, that looks generated. Also it is an example and has more problems as we already changed some other metrics from that file.

**Pull request merged**